config/rbac/manual_role.yaml

```diff
@@ -0,0 +1,223 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  creationTimestamp: null
+  name: manual-manager-role
```

Comment:
The name is a bit confusing. What about skipping role generation altogether at the moment, and keep the original role manifest with the extra rules added manually?

Reply:
That's what I did at first, but it's failing CI since it checks against generated files and also regenerates files before running tests.

Reply:
Ah right, it wasn't generating anything before because there were no kubebuilder markers.

What about naming it `mead-manager-role`? I think that's more descriptive.